app/utilities/schema.py

```diff
@@ -36,10 +33,10 @@ def load_and_parse_schema(eq_id, form_type):
     """
     # load the schema
 
-    json_schema = load_schema(eq_id, form_type)
+    json_schema = load_schema(eq_id, form_type, language_code)
     if json_schema:
         parser = SchemaParserFactory.create_parser(json_schema)
         schema = parser.parse()
         return json_schema, schema
     else:
-        return None
+        return None, None
```

Comment:
Not in the scope of this change but load_schema will fail and raise either FileNotFoundError or botocore.exceptions.ClientError if it fails to find a schema, it will be caught logged and then None returned. I wonder if we should change that going forward to just raise the error log it and reraise it?